dotnet/samples/PlayFabExamples/Example02_Generative/SegmentOpenAPIs.json

```diff
@@ -0,0 +1,1685 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "PlayFab APIs [Sample]",
+    "termsOfService": "https://playfab.com/terms/",
+    "contact": {
+      "url": "https://community.playfab.com/index.html"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://github.com/PlayFab/API_Specs/blob/master/LICENSE"
+    },
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://4B51D88A.data-svcs.playfabapi.com",
```

Comment:
Is there a way to pass in the TitleId and the endpoint via config rather than have it hard-coded here?

Reply:
I will replace this with https://<titleid>.playfabapi.com. This shouldn't impact. We are not using this value.